packages/bot-web-ui/src/stores/dashboard-store.ts

```diff
@@ -1,18 +1,25 @@
 import { observable, action } from 'mobx';
 import RootStore from './root-store';
+import { removeExistingWorkspace } from '@deriv/bot-skeleton';
 
+const clearInjectionDiv = () => {
```

Comment:
We have written this function more than once. We better transfer it to a shared file like utility and use it multiple times.

Reply:
done ✅